script/Arcueid/Day 13/13_00_ARC13_5.txt

```diff
@@ -396,12 +400,12 @@ It doesn't matter how far away from me he is anymore.
 [sha:9213a3084dd23661223dff8bc436f852c34e6cb3]{
 -- Page 426, Offset 32890.
 -- 俺より前にいればそれでいい。
-So long as he's in front of me, then it's fine.
+It's enough that he's in front of me.
 }
 [sha:61432f1c51c319cd8b6fc2dbfa9b2ad95758739e]{
 -- Page 426, Offset 32891.
 -- ただ、<生|・><き|・><て|・><さ|・><え|・><い|・><れ|・><ば|・>、それでいい。
-%{i}As long as I'm still alive%{/i}, I can make it work.
+It's enough that %{i}he's simply alive%{/i}.
```

Comment:
Kinda hard to perfectly express this one in EN, but not sure I love "he's simply alive". Ideally it'd be more clear that the reading is "if he's something that has life [and can therefore be robbed of it] then it's all good". 

> All that matters is that he's in front of me.
> All that matters is that he's %{i}alive%{/i}.